src/main/java/io/confluent/connect/jdbc/TableQuerier.java

```diff
@@ -107,4 +121,11 @@ public int compareTo(TableQuerier other) {
       return this.name.compareTo(other.name);
     }
   }
+
+  private boolean exceptionIndicatesStalePreparedStatement(Connection db, SQLException e) throws SQLException {
+    // On PostgreSQL, queries with wildcards will be invalidated on schema changes
+    // and throw this error message.
+    return e.getMessage().contains("cached plan must not change result type") &&
```

Comment:
Actually that probably won't work, because PSQLException isn't in scope of this library unless we want to add that dependency